packages/filefeeds-react/src/OneSchemaFileFeeds.tsx

```diff
@@ -178,6 +183,7 @@ export default function OneSchemaFileFeeds({
 
     instance?.on("destroyed", (data: DestroyedEventData) => {
       onDestroy?.(data)
+      onRequestClose?.()
```

Comment:
should we call onrequest close first here? or after

Reply:
This is the order we do this for all other cases. Let's keep it like this for now until we have a more clear idea about what destroy event means here, and whether we like to keep it around.